<launch>
	<!-- run the landpad_det.cpp -->
    <node pkg="auto_landing" type="landpad_det" name="landpad_det" output="screen">
        <param name="camera_topic" type="string" value="/prometheus/sensor/monocular_down/image_raw" />
        <param name="camera_info" type="string" value="$(find prometheus_gazebo)/config/camera_config/camera_param_gazebo_monocular.yaml" />
    </node>
</launch>
